### PR TITLE
test: Update openshift image for ansible and kubeconfig

### PR DESCRIPTION
**Pull request by stefwalter**
Setup /var/lib/kubelet/kubeconfig correctly on openshift image.
Also make ansible able to work on these images.

Put in some config for AuthorizedKeysCommand  used during
development. Only used when available.

### CODE REVIEW

**Comment by stefwalter** [comment]
Uploaded the image

File: test/images/scripts/openshift.setup [hunk 1]
```diff
@@ -55,7 +55,7 @@ chmod +x /openshift-prep
 touch /openshift-run
 chmod +x /openshift-run
 
-printf '[Unit]\nDescription=Openshift\nWants=network-online.target\nAfter=network-online.target\n[Service]\nExecStartPre=/openshift-prep\nExecStart=/openshift-run\n[Install]\nWantedBy=multi-user.target\n' > /etc/systemd/system/openshift.service
+printf '[Unit]\nDescription=Openshift\nWants=network-online.target\nAfter=network-online.target\n[Service]\nExecStartPre=/openshift-prep\nExecStart=/openshift-run\nRestart=always\nRestartSec=60\n[Install]\nWantedBy=multi-user.target\n' > /etc/systemd/system/openshift.service
```
Comment (stefwalter) [inline test/images/scripts/openshift.setup]: The Restart changes here help tests be less racy. Fingers crossed.

Comment (dperpeet) [inline test/images/scripts/openshift.setup]: This has become pretty difficult to read. Can you put this in a file (those are uploaded) or make it multiline?

File: test/images/scripts/openshift.setup [hunk 2]
```diff
@@ -180,6 +180,13 @@ cat /root/.kube/config
 # Wait a bit in case an operator wants to copy some info
 sleep 20
 
+# Use standard locations for kubelet kubeconfig
+mkdir -p /var/lib/kubelet
+cp /openshift.local.config/node-f1.cockpit.lan/node.kubeconfig /var/lib/kubelet/kubeconfig
```
Comment (dperpeet) [inline test/images/scripts/openshift.setup]: should we comment on where "node-f1" comes from?

**Comment by petervo** [comment]
I can do this in a followup if you are not regenerating, but I think we might need to add

```systemctl enable systemd-networkd-wait-online```

to the setup script.

**Comment by stefwalter** [comment]
> I can do this in a followup if you are not regenerating, but I think we might need to add

I need to regenerate to fix @dperpeet's review

**Pull request closed without merging**